docs/_articles/en/v3.x/salesforce-graph-engine/working-with-sfge.md

```diff
@@ -122,6 +122,7 @@ Graph Engine has these limitations.
 * Graph Engine doesn’t handle anonymous Apex script. Provide the class directory path as the `--projectdir` that doesn’t include any anonymous Apex script.
 * Graph Engine doesn’t handle namespace placeholders. Leave the namespace placeholder blank.
 * Graph Engine supports Apex property chains with a depth of 2 or fewer. For example, Graph Engine supports `Object.x` but not `Object.x.y`.
+* Graph Engine scans entry points, but it doesn't scan triggers.
```

Comment:
This implies that triggers aren't entry points. They are, they're just not entry points that we currently support.